### DISCUSSION TITLE
BreadcrumbItem 与 ICon 渲染 html 节点处理方式

### DISCUSSION THREAD

**Discussion by wuxinheng** [AchievedOwner/TDesignBlazor#322] [讨论]
TBreadcrumbItem文档用法：
`<TBreadcrumbItem><TIcon Name="IconName.Books"/> 文案</TBreadcrumbItem>`

TIcon 从用户的角度 TBreadcrumbItem 编写在 TBreadcrumbItem 标签内是正确的，但是从前端排版的角度来看渲染标签却需要拆分将文本类容与Icon同级处理，独立出来，现在的处理方式：

```
<p style="max-width:150px;white-space: nowrap;overflow: hidden;text-overflow: ellipsis;">
选择列表<i cass="t-icon t-icon-view-list"></i>
</p>
```

现在发现的问题：icon 垂直居中难以调整、甚至会影响其他样式

不易于排版，样式之间容易相互影响，我认为起码应该这样处理

```
<p style="max-width:150px;white-space: nowrap;overflow: hidden;text-overflow: ellipsis;">选择列表</p>
<i class="t-icon t-icon-view-list"></i>
```

**Comment by teacher-zhou** [comment]
感觉这是官方的样式的问题造成的
